metrics/metric_java.py

```diff
@@ -0,0 +1,38 @@
+from sqlalchemy.orm import Query
+
+from metrics.metric_common import MetricCommon
+from models.metric import Metric
+from models.version import Version
+
+class MetricJava(MetricCommon):
+
+    def __init__(self, session, config):
+        super(MetricJava, self).__init__(session, config)
+
+        self.ck_metrics_query = self._get_ck_metrics_query().subquery()
+        self.query = self.session.query(
```

Comment:
It should be better to have a method to do all the operation
Maybe a method called get_langage_specific_metrics, called by the parent class

Reply:
Done